import_export/management/commands/export.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand
+
+from import_export.command_utils import (
+    get_default_format_names,
+    get_format_class,
+    get_resource_class,
+)
+
+
+class Command(BaseCommand):
+    help = "Export data from a specified resource or model in a chosen format."
```

Comment:
one thing I noticed when testing this:

```bash
./manage.py export XLSX core.tests.resources.BookResource
```

This dumps a load of binary to stdout.  Apparently it's ok to export binary data to the terminal, but it can mess up the [terminal output](https://askubuntu.com/a/1105350).  What do you think?  Perhaps it just needs a comment somewhere to say that binary data can be output

Reply:
Agree -  now it is checked and error is raised if terminal is atty. Added in:
 
https://github.com/django-import-export/django-import-export/pull/1992/commits/cc4b4b72a9dd40bef7a44f683641e99b0d82a7b6